package.json

```diff
@@ -50,7 +50,7 @@
     "typescript": "^3.6.0-dev.20190804"
   },
   "peerDependencies": {
-    "typescript": ">=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta"
+    "typescript": ">=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta || >= 4.0.2 "
```

Comment:
why is this even necessary? `>=2.8.0` includes 4.x as well

Reply:
I dont know but without the change it was not working for me.